package.json

```diff
@@ -29,9 +29,10 @@
     "fs-extra": "^8.1.0",
     "lodash": "^4.17.11",
     "loopback-connector": "^4.0.0",
-    "loopback-connector-couchdb2": "^1.5.1",
+    "loopback-connector-couchdb2": "github:strongloop/loopback-connector-couchdb2#add-option-for-search",
```

Comment:
Don't forget to switch back to npm released version.

Reply:
👍 sure